RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.14.0
+
+- added: `push/rebase=yes/no` boolean branch annotation qualifiers that control rebase/push behaviour of the `git machete traverse` command
```

Comment:
Plus also `merge` ;)

Reply:
Maybe write as
```suggestion
- added: `{push,rebase,merge}={yes,no}` boolean branch annotation qualifiers that control rebase/push behaviour of the `git machete traverse` command
```
for consistency with the _brace expansion_ syntax that we use e.g. in docs

Reply:
Also... `yes` is only supposed to be supported for `merge`, right? not for `push` and `rebase`? 🤔

Reply:
Right, `yes` is only supposed to be supported for `merge`